### ISSUE TITLE
Not opening all of a split request

### ISSUE THREAD

**Issue by lmullane** [bcgov/foi-flow#131]
* As an analyst
* I want to be able to split a request but only open some of the requests that have been split
* so that I can open a request for Ministry A and get clarification on the request for ministry B

**Assumptions & Scope**	
What are the assumptions for this story?

What is in and not in scope?

**Acceptance Criteria**	

**Scenario 1: xxxxxx**
* GIVEN ... (you or your condition)
* WHEN ... (what you do) 
* THEN ... (what you see) 

**Scenario 2: xxxxxx**
* GIVEN ... (you or your condition)
* WHEN ... (what you do) 
* THEN ... (what you see) 

**Scenario 3: xxxxxx**
...

**Dependencies? What is the impact of this dependency? (If so, link dependency in the ticket, make it visible in a team´s backlog)**

**Validation Rules? (If yes, list here)**	

**Design**
@xxx - please link the Design here

**Definition of Ready**

1. [ ] Is there a well articulated User Story?
2. [ ] Is there Acceptance Criteria that covers all scenarios (happy/sad paths)?
3. [ ] If there is a user interface, is there a design? 
4. [ ] Does the user story need user research/validation?
5. [ ] Does this User Story needs stakeholder approval?
6. [ ] Design / Solution accepted by Product Owner
7. [ ] Is this user story small enough to be completed in a Sprint? Should it be split?
8. [ ] Are the dependencies known/ understood? (technical, business, regulatory/policy)
9. [ ] Has the story been estimated?

**Definition of Done**

1.    [ ] Passes developer unit tests
2.    [ ] Passes peer code review
3.    [ ] If there's a user interface, passes UX assurance
4.    [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
5.	[ ] Confirm Test cases built and succeeding 
6.	[ ] No regression test failures
7.	[ ] Test coverage acceptable by Product Owner
8.	[ ] Ticket ready to be merged to master or story branch
9.	[ ] Developer to list Config changes/ Update documents and designs
10.	[ ] Can be demoed in Sprint Review 
11.	[ ] Tagged as part of a Release
12.   [ ] Feature flagged if required
13.  [ ] Change Management activities done?

**Comment by JHarrietha-AOT** [comment]
@mpilchar @lmullane There is no acceptance criteria to review

**Comment by lmullane** [comment]
Hmmm...thought I did this one already. Will draft now.

**Comment by lmullane** [comment]
@JHarrietha-AOT, let's replace the 'Save the Request Description" button with one universal 'Save' button that when clicked will save all changes made by the analyst.

Could we update the designs accordingly? cc @mpilchar

**Comment by JHarrietha-AOT** [comment]
@lmullane Updated the hotspots looked like they shifted on these split screens. The above should be resolved let me know if you are still have issues. I will update the save button before stand up tomorrow.

**Comment by lmullane** [comment]
Sorry, still not working. When I click 'Split Request' I get a blank screen

**Comment by JHarrietha-AOT** [comment]
@lmullane That's so weird, its working for me. I shared the link with @arielleandrews to see if its working for her
https://jacklyn808742.invisionapp.com/console/share/3W2O64AP5C/663873013

**Comment by lmullane** [comment]
Ah, I think I know what's up here. The design covers the split a request for a redirect (BC Hydro) and open a request (FNR), correct?

What about clarifying part of the request (i.e. opening FNR but not EML). That was what I was looking for, but thinking the design is not for that use case?

**Comment by arielleandrews** [comment]
I can get to the Split Request screen. From there I can hit the Clarification/Redirect toggle and then select EML in the list of ministries. But if I try to select EML first then I get a blank screen

**Comment by JHarrietha-AOT** [comment]
@lmullane You should be able to click on EML now, you should be able to in this use case. I just re-uploaded that screen and looks like its working now can you and @arielleandrews let me know if its not

**Comment by arielleandrews** [comment]
It's working for me @JHarrietha-AOT

**Comment by lmullane** [comment]
@JHarrietha-AOT, should be good now. Needed to toggle and select EML in the right order. 

Please review the AC:)

**Comment by lmullane** [comment]
@JHarrietha-AOT, just want to be sure you saw my comment about removing the 'Save Description' button with a universal 'Save' button?

**Comment by JHarrietha-AOT** [comment]
@lmullane for Scenario 3 - I don't think we would want redirected requests to automatically be closed. Please correct me if I am wrong @arielleandrews. I remember intake saying there are times when applicants respond to the redirect letter with clarifications that can prompt the opening of that request, as the applicant my not have been clear with what they actually wanted.

Also, I am working on changing it to a universal save button, will have it updated by stand-up tomorrow, or sooner.

**Comment by lmullane** [comment]
Cool, I'll say they go into Redirect Status

**Comment by JHarrietha-AOT** [comment]
Sounds good! Other then that all AC is good

**Comment by arielleandrews** [comment]
Yes @JHarrietha-AOT and @lmullane having the request go to Redirect status makes sense. I would say functionally this would be similar to Closed in that the required action has been taken by informing the applicant they need to resubmit their request elsewhere. But as we laid out in the Statuses diagram if the applicant comes back and it turns out we can actually proceed with their request, that we can then take the request back from Redirect through to Open

**Comment by abin-aot** [comment]
### Backlog refinement queries

1.  Do we need a "Date Range for record search" for Redirection ? If yes, could you please clarify.

2. Seems like I got lost on the links here and there, but referring to [https://jacklyn808742.invisionapp.com/console/FOI---Intake-ckpq19frj032r013q6gpm8t1m/ckpx6mtxp012t013e4wg0agtg/play](https://jacklyn808742.invisionapp.com/console/FOI---Intake-ckpq19frj032r013q6gpm8t1m/ckpx6mtxp012t013e4wg0agtg/play) , Seems like more of the scenarios are technically/foundation part covered under User Story #129, other than redirection. Right?

**Comment by lmullane** [comment]
The original date range can go with the Redirect.
